src/components/graph-view.js

```diff
@@ -343,7 +343,7 @@ class GraphView extends Component {
       if(!self.props.readOnly){
         var d = d3.select(this).datum();
         // Move the node back to the original z-index
-        if (oldSibling) {
+        if (oldSibling && oldSibling.parentElement) {
```

Comment:
I think before we blindly check if `oldSibling` has a parent element (because it should always have one if it's not null), we should find out why we got a non-null value for `oldSibling` that ended up not having a parent element. What was this truthy object and how did it get there? Perhaps `this.nextSibling` is the cause of the problem because it's only used once, never defined, and that's where it defines `oldSibling`.